specs/spec-000-rewrite/spec.md

```
@@ -0,0 +1,30 @@
+<!-- SPDX-License-Identifier: Apache-2.0 OR MIND-UCAL-1.0 -->
+<!-- © James Ross Ω FLYING•ROBOTS <https://github.com/flyingrobots> -->
+# SPEC-000: Everything Is a Rewrite
+
+**Status:** Draft (live demo scaffold)
+
+## Purpose
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)